src/sentry/models/release.py

```diff
@@ -132,6 +134,20 @@ def annotate_prerelease_column(self):
             )
         )
 
+    def annotate_adoption_date_column(self):
+        from sentry.models import ReleaseProjectEnvironment
+
+        return self.annotate(
+            adopted=Coalesce(
+                Subquery(
+                    ReleaseProjectEnvironment.objects.filter(release_id=OuterRef("pk"))
```

Comment:
This should probably be filtered by project/env if they are selected. Otherwise we might end up sorting by a date from a different `project/env` then is selected.

Reply:
Aah you're right, thanks! Good catch